change/@fluentui-react-tabster-57b7754a-68fe-4e13-b419-cc877b452a24.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "none",
```

Comment:
hmm this looks wrong, it should by type:feat with minor bump as the bump is adding new features